code/modules/mob/living/life.dm

```diff
@@ -88,6 +91,12 @@
 /mob/living/proc/handle_stomach()
 	return
 
+/mob/living/proc/handle_ambience() // If you're in an ambient area and have not moved out of it for x time, we're going to play ambience again to you, to help break up the silence.
+	if(world.time >= (lastareachange + 30 SECONDS)) // Every 30 seconds, we're going to run a 35% chance to play ambience.

```

Comment:
Is the 35% in play_ambience()?
Does play_ambience() update lastareachange, or will this fire every life tick after the first 30 seconds?

Reply:
If the prob(35) in the play_ambience() fires, it will update the last-played - so while yes this fires every life tick afterwards until you move to another /area/, you realistically cannot get ambience played to you any longer than once per 60 seconds, and even then, it's a 35% chance to have the ambience played.

Reply:
So yes, the 35% is in play_ambience()